hyperopt/tests/test_rdists.py

```diff
@@ -152,6 +155,13 @@ def test_offgrid_float(self):
         assert qn.pmf(.99) == 0.0
         assert qn.pmf(-.99) == 0.0
 
+    def test_output_type_int(self):
+        result = quniform_gen(0, 10, 1).rvs()
```

Comment:
One minor detail: is there a reason why you assign the thing you want to test to `result` in the `*_int` tests but 
test it directly in the `*_float` ones?